### PR TITLE
Migrate `desktop_photo_search` to top level

### PR DESCRIPTION
**Pull request by domesticmouse**
## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: desktop_photo_search/fluent_ui/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  fluent_ui: ^3.7.0
+  fluentui_system_icons: ^1.1.157
+  flutter:
+    sdk: flutter
+  http: ^0.13.4
+  logging: ^1.0.2
+  menubar:
+    git:
+      url: https://github.com/google/flutter-desktop-embedding.git
+      path: plugins/menubar
```
Comment (RedBrogdon) [inline desktop_photo_search/fluent_ui/pubspec.yaml:26]: Is this something that'll get published in the future?

File: desktop_photo_search/fluent_ui/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  msix: ^2.8.5
+
+flutter:
+  uses-material-design: true
+
+msix_config:
+  display_name: Flutter Desktop Photo Search
+  publisher_display_name: flutter.dev
+  store: false # Set to true to deploy to Microsoft Store
+  publisher: CN=01A6D5C0-D51A-4EEE-8DD0-F134DDD378F7
```
Comment (RedBrogdon) [inline desktop_photo_search/fluent_ui/pubspec.yaml:49]: Just checking: is it cool for this GUID to be public?

**Review by RedBrogdon — APPROVED** [review]
LGTM, but hold off on landing this until #1006 lands.

File: desktop_photo_search/fluent_ui/lib/src/widgets/photo_details.dart [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+class _PhotoDetailsState extends State<PhotoDetails> {
+  Widget _buildPhotoAttribution(BuildContext context) {
+    return Row(
+      children: [
+        const Text('Photo by'),
+        Link(
+          uri: Uri.parse(
+              'https://unsplash.com/@${widget.photo.user!.username}?utm_source=$unsplashAppName&utm_medium=referral'),
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply desktop_photo_search/fluent_ui/lib/src/widgets/photo_details.dart:39]: It's required by the API usage spec 😎

File: desktop_photo_search/fluent_ui/pubspec.yaml (hunk 1 above)
Comment (domesticmouse) [reply desktop_photo_search/fluent_ui/pubspec.yaml:26]: There is ongoing work on design of the API for menus.

File: desktop_photo_search/fluent_ui/pubspec.yaml (hunk 2 above)
Comment (domesticmouse) [reply desktop_photo_search/fluent_ui/pubspec.yaml:49]: yup

File: desktop_photo_search/fluent_ui/lib/src/widgets/policy_dialog.dart [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                TextSpan(
+                  text: 'https://policies.google.com/terms',
+                  style: TextStyle(
+                      fontWeight: FontWeight.bold, color: Colors.blue.normal),
+                  recognizer: TapGestureRecognizer()
+                    ..onTap = () async {
+                      const url = 'https://policies.google.com/terms';
+                      if (await url_launcher.canLaunch(url)) {
+                        await url_launcher.launch(url);
+                      }
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply desktop_photo_search/fluent_ui/lib/src/widgets/policy_dialog.dart:35]: There should really be an else here with an altert or the like.

**Pull request merged**